pubsub/rabbitmq/rabbitmq.go

```diff
@@ -46,12 +46,17 @@ const (
 	publishRetryWaitSeconds = 2
 
 	argQueueMode          = "x-queue-mode"
-	argMaxLength          = "x-max-length"
+	argMaxLength          = "x-max-length" // only for quorum and classic queues
 	argMaxLengthBytes     = "x-max-length-bytes"
+	argMaxAge             = "x-max-age" // only for stream type queues
```

Comment:
Should we rename parameter name to reflect that it is for stream specifically? Something like, StreamMaxAge.

Just thinking, if we should use directly amqp constant here as well? 
https://github.com/rabbitmq/amqp091-go/blob/09d223a6803ba6824f3c311c82a3f630c1b162a9/types.go#L255